deblur/deblurring.py

```diff
@@ -148,7 +149,7 @@ def deblur(input_seqs, read_error=0.05, mean_error=None, error_dist=None,
             mask = (sub_seq_i != sub_seq_j)
             muttype = np.logical_or(sub_seq_i[mask] == 4, sub_seq_j[mask] == 4)
             num_indels = muttype.sum()
-            num_substitutions = len(muttype) - num_indels
+            num_substitutions = h_dist - num_indels
```

Comment:
I don't think this change is correct - and if it is, then the name of the variables is misleading. Currently this code is doing the hamming distance minus the num of indels. Isn't that conceptually different that counting the number of substitutions as it was done before?